### ISSUE TITLE
Error in test: DailySolarRadiation

### ISSUE THREAD

**Issue by fong22e** [End-to-end-provenance/RDataTracker#485]
I am getting this error when I am running the regression test. Does anyone know where this could be from and how I should fix this?
```
     [echo]
     [echo]     Diff of (1) DailySolarRadiation.out.local and (2) expected_DailySolarRadiation.out.local:
     [echo]
     [exec] 1,10c1
     [exec] < Error in file(file, ifelse(append, "a", "w")) :
     [exec] <   cannot open the connection
     [exec] < Calls: prov.run ... .ddg.data.node -> .ddg.save.snapshot -> write -> cat -> file
     [exec] < In addition: Warning message:
     [exec] < In file(file, ifelse(append, "a", "w")) :
     [exec] <   cannot open file '[DIR]/prov_DailySolarRadiation/data/64-plot.data(raw.data,"R") return.txt': Invalid argument
     [exec] < Execution halted
     [exec] < Warning message:
     [exec] < In .ddg.insert.error.message("Start and finish nodes do not match") :
     [exec] <   Start and finish nodes do not match
     [exec] ---
     [exec] >
     [exec] Result: 1
     [exec] scriptTests/DailySolarRadiation/local/DailySolarRadiation.out.local scriptTests/DailySolarRadiation/local/expected_DailySolarRadiation.out.local differ: byte 1, line 1
     [exec] Result: 1
```

**Comment by blernermhc** [comment]
Is this testing RDataTracker or provR?

Also, what happens if you just run the tool on the test, without using the testing script?

**Comment by fong22e** [comment]
RDataTracker. The test runs fine when run in RStudio on using both RDataTracker and provR. When I just run the test itself using ant without using the `test-travis` target, the same error is thrown.

**Comment by blernermhc** [comment]
I do not see this behavior on the development branch.

**Comment by blernermhc** [comment]
Are we hitting a limit on the length of a file name on Windows?  If you look in the .out file in the prov directory (not the local directory), how long is the full path to the file?

If you run from RStudio, is the provenance stored in a directory that results in a shorter file name?

**Comment by fong22e** [comment]
When I run from RStudio and have the working directory set to the directory of where the test is, this problem does not occur.

**Comment by fong22e** [comment]
I thought it was mysteriously fixed but I was wrong. The problem is still there.

**Comment by blernermhc** [comment]
Just to be clear, with the RStudio testing and the ant testing, is the provenance being written to the same directory, so the filename length should be the same for both?

**Comment by fong22e** [comment]
If I’m correct in thinking that the ant testing will write to the prov directory (prov_DailySolarRadiation), then yes, both the RStudio testing and ant testing writes to the same directory.

**Comment by fong22e** [comment]
Unfortunately, I am still having this error. I have merged in development and master today.

**Comment by blernermhc** [comment]
I have checked out the provRtests branch, installed a version of rdt from it, and tested DailySolarRadiation, and I do not get this error.
```
ant -file build-rdt.xml install
ant -file tests.xml DailySolarRadiation-rdt
```
Are those the commands that I should be using to try to reproduce it?

**Comment by fong22e** [comment]
Yes, those commands are correct. That is very strange. I am still having the same issue.

**Comment by fong22e** [comment]
As a test, I changed the test name to "Solar" and I still have the same problem.

**Comment by fong22e** [comment]
Could this be a Windows issue? Emery ( @erboose ), do you have this issue?

**Comment by erboose** [comment]
Which branch?

**Comment by fong22e** [comment]
This test works on Linux (Travis) and on Mac, but fails on Windows.
